tools/pythonpkg/src/native/python_conversion.cpp

```diff
@@ -453,8 +453,8 @@ Value TransformPythonValue(py::handle ele, const LogicalType &target_type, bool
 	case PythonObjectType::Datetime: {
 		auto &import_cache = *DuckDBPyConnection::ImportCache();
 		bool is_nat = false;
-		if (import_cache.pandas().isnull.IsLoaded()) {
-			auto isnull_result = import_cache.pandas().isnull()(ele);
+		if (import_cache.pandas.isnull(false)) {
```

Comment:
This behavior isn't entirely equivalent?
The old version loaded `pandas` entirely, caching all of the sub attributes we care about, then try to see if `isnull` was present.
If it's present in the installed version of Pandas it would be loaded

The new behavior does not try to load anything, instead it merely checks if another path has been used that loaded `isnull` before